File: src/main/java-graphql/io/github/graphql/java/model/MoveProjectCardMutationResponse.java
package io.github.graphql.java.model;

import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLResult;
import java.util.Map;

@javax.annotation.Generated(
    value = "com.kobylynskyi.graphql.codegen.GraphQLCodegen",
    date = "2021-03-07T19:46:14+0800"
)
public class MoveProjectCardMutationResponse extends GraphQLResult<Map<String, MoveProjectCardPayloadTO>> {

    private static final String OPERATION_NAME = "moveProjectCard";

    public MoveProjectCardMutationResponse() {
    }

    public MoveProjectCardPayloadTO moveProjectCard() {
        Map<String, MoveProjectCardPayloadTO> data = getData();
        return data != null ? data.get(OPERATION_NAME) : null;
    }

}

File: src/main/scala-graphql/io/github/graphql/scala/model/MoveProjectCardMutationResponse.scala
package io.github.graphql.scala.model

import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLResult
import java.util.{ Map => JMap }

@javax.annotation.Generated(
    value = Array("com.kobylynskyi.graphql.codegen.GraphQLCodegen"),
    date = "2021-03-07T19:49:37+0800"
)
class MoveProjectCardMutationResponse extends GraphQLResult[JMap[String, MoveProjectCardPayloadTO]] {

    def moveProjectCard(): MoveProjectCardPayloadTO = {
        val data: JMap[String, MoveProjectCardPayloadTO] = getData
        if (data != null) data.get(MoveProjectCardMutationResponse.OPERATION_NAME) else null
    }

}

object MoveProjectCardMutationResponse {

    private final val OPERATION_NAME: String = "moveProjectCard"

}
